### ISSUE TITLE
Down arrow on writing makes text disappear

### ISSUE THREAD

**Issue by txomon** [glowing-bear/glowing-bear#970]
I have been using glowing-bear for a while, and I find difficult to know which are the key bindings. In specific, the one that is annoying me is that when writing, down arrow makes the text disappear.

I don't know if this is intended or a side effect, but if there was a place with the keybindings listed and behaviours, that would be quite useful. Also, taking out this behaviour would also be nice.

**Comment by lorenzhs** [comment]
Hi @txomon, the keybindings are listed on the landing page under "Usage instructions". Arrow keys navigate the history (implemented in #95), and pressing down is a way to push the current line into the history stack so you can get it back later. We're just mimicking WeeChat's behaviour. It's quite powerful when you know how it works, you can push a draft into the history if you just want to fire off a quick message in reply to someone else without having to bother with the clipboard, or being limited to one saved message at a time. The code that's responsible for this is at https://github.com/glowing-bear/glowing-bear/blob/84018d4bec3d17cd9edafcd55843054959b9d992/js/models.js#L258

I agree that it's not very discoverable to someone who doesn't already know it from WeeChat. Any thoughts on how we could improve that?

tl;dr that's by design

**Comment by txomon** [comment]
Oh ok, I haven't logged in in a long long while (is one of those pinned tabs I have). Would it be possible to have it somewhere, the usage instructions, like for example Gmail / Inbox / Vimfx (firefox) / Vimium (Chrome) do?

An overlay or a help icon is good enough

**Comment by txomon** [comment]
Also, I have noticed that when unfolding the usage instructions, I am unable to login

![image](https://user-images.githubusercontent.com/228250/31661248-eea4fdbe-b331-11e7-80d0-1bc84af320a6.png)

**Comment by lorenzhs** [comment]
The unfolding thing should be fixed in https://latest.glowing-bear.org.

A help overlay on pressing '?' would certainly be nice, good idea!

**Comment by txomon** [comment]
It is now folding the unfolded sections, just in case. Also, the history
push you mentioned is not in the usage instructions, you may want to add
that too..

>
